RELEASE.md

```diff
@@ -1,3 +1,8 @@
+# Release 1.0.10
+
+- Updates doc to avoid all use of `http:`
```

Comment:
This work makes changes to doc as well as source code. As such, I suggest we modify this line to read,

```suggestion
- Updates project to favor the use of secured `https` rather than `http`
```

Reply:
updated